packages/components/src/components/shell-panel/shell-panel.browser.e2e.tsx

```diff
@@ -0,0 +1,101 @@
+import { h } from "@arcgis/lumina";
+import { mount } from "@arcgis/lumina-compiler/testing";
+import { it, expect, describe, beforeEach, afterEach, vi } from "vitest";
+import "../shell/shell";
+import "./shell-panel";
+import "../panel/panel";
+import { CSS } from "./resources";
+import type { ShellPanel } from "./shell-panel";
+
+describe("shell-panel updateSize public method", () => {
+  // mock known Lit nested menu warn so failOnConsole doesn’t fail
+  let warnSpy: ReturnType<typeof vi.spyOn>;
+
+  beforeEach(() => {
+    warnSpy = vi.spyOn(console, "warn").mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    warnSpy.mockRestore();
+  });
+  it("should update default vertical size via token, manual resize, method, and reset to token", async () => {
+    const initialToken = 320;
+    const methodResize = 400;
+
+    const { el } = await mount<"calcite-shell">(
+      <calcite-shell>
+        <calcite-shell-panel
+          resizable
+          slot="panel-start"
+          style={`--calcite-shell-panel-width: ${initialToken}px;`}
+        >
+          <calcite-panel>Content</calcite-panel>
+        </calcite-shell-panel>
+      </calcite-shell>,
+    );
+
+    const panel = el.querySelector("calcite-shell-panel") as ShellPanel;
+    expect(panel).toBeTruthy();
+
+    const content = panel.shadowRoot.querySelector(`.${CSS.content}`) as HTMLElement;
+    const handle = panel.shadowRoot.querySelector(`.${CSS.resizeHandle}`) as HTMLElement;
+    expect(content).toBeTruthy();
+    expect(handle).toBeTruthy();
+
+    expect(getComputedStyle(content).width).toBe(`${initialToken}px`);
+
+    handle.dispatchEvent(new KeyboardEvent("keydown", { key: "ArrowRight", bubbles: true }));
```

Comment:
Use [`userEvent`](https://vitest.dev/api/browser/interactivity.html#userevent-keyboard) to simulate user interactions (see [example](https://github.com/Esri/calcite-design-system/blob/dev/packages/components/src/components/radio-button-group/radio-button-group.browser.e2e.tsx#L91-L92)).